cluster-up/cluster/kind-k8s-sriov-1.17.0/provider.sh

```diff
@@ -25,7 +20,7 @@ function up() {
 
     kind_up
 
-    # remove the rancher.io kind default storageClass
+     remove the rancher.io kind default storageClass
```

Comment:
The comment was removed?

Reply:
No, just a mistake
I will bring it back